pages/osx/afinfo.md

```diff
@@ -0,0 +1,24 @@
+# afinfo
+
+> Audio file metadata parser for OS X.
```

Comment:
Is there any document link we can add here?

Reply:
This is a built-in command provided by OS X, I have updated the description.